docs/UserGuide.md

```diff
@@ -167,13 +167,28 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/ROOM_NUMBER] [a/ADDRESS] [en
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Should the INDEX be specified within the "Note" block as well, since the "Note" block talks about the other field constraints? I feel like the way I did it for `add` isn't good either though cause everything is in the "Note" block which kind of defeats the purpose. Do you think theres a better way to format it?